docsets/Faker.py/docset.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Faker",
+  "version": "19.6.2",
+  "archive": "Faker.tgz",
+  "author": {
+    "name": "Xavier Yang",
+    "link": "https://github.com/ivaquero"
+  },
+  "aliases": ["Faker", "Python", "Python Faker"],
+  "specific_versions": [
+    {
+      "version": "19.6.2",
+      "archive": "versions/Faker.tgz"
```

Comment:
This path doesn't exist.

Reply:
Oh, that's a mistake. I will solve it later.